daemon/daemon.go

```diff
@@ -442,7 +442,17 @@ configRetry:
 		log.Infof("Starting the Typha connection")
 		err := typhaConnection.Start(context.Background())
 		if err != nil {
-			log.WithError(err).Fatal("Failed to connect to Typha")
+			retry := 0
+			for err != nil && retry < 10 {
+				// Set Ready and Live to false
+				healthAggregator.Report(healthName, &health.HealthReport{Live: false, Ready: false})
```

Comment:
I think this should be `Live:true, Ready:false`  If we report non-live then Kubernetes will kill Felix but in this case we're managing the retry and restart ourselves.

Reply:
Ack